frontend/app/services/subscription-service.server.ts

```diff
@@ -136,6 +136,39 @@ function createSubscriptionService() {
     return cdcpSubscriptions.at(0);
   }
 
+  async function deleteSubscription(userId: string, subscriptionId: string) {
+    const auditService = getAuditService();
+    const instrumentationService = getInstrumentationService();
+    auditService.audit('alert-subscription.delete', { subscriptionId });
+
+    const url = new URL(`${CDCP_API_BASE_URI}/api/v1/users/${userId}/subscriptions/${subscriptionId}`);
+
+    const response = await fetch(url, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    instrumentationService.countHttpStatus('http.client.cdcp-api.alert-subscription.delete', response.status);
+
+    if (response.status === 204) {
+      return null;
+    }
```

Comment:
No need to return anything from this function if deleting is successful.

Reply:
removed